pkg/proxy/ipvs/proxier.go

```diff
@@ -273,6 +273,12 @@ type Proxier struct {
 	// Inject for test purpose.
 	networkInterfacer     utilproxy.NetworkInterfacer
 	gracefuldeleteManager *GracefulTerminationManager
+	// localNoEndpoints represents the number of rules that couldn't be applied due to
+	// the absence of local endpoints.
+	localNoEndpoints int64
+	// externalNoEndpoints represents the number of rules that couldn't be applied due to
+	// the absence of any endpoints.
+	externalNoEndpoints int64
```

Comment:
nit: I would name this clusterNoEndpoints instead

Reply:
Done